metaflow/cli.py

```diff
@@ -649,16 +649,39 @@ def run(obj,
               default=DEFAULT_RUN_NAME,
               help="name assigned to the new KFP run"
               )
+@click.option('--namespace',
+              'namespace',
+              default=KFP_SDK_NAMESPACE,
+              help="namespace of your run in KFP."
+              )
+@click.option('--api-namespace',
+              'api_namespace',
+              default=KFP_SDK_API_NAMESPACE,
+              help="namespace where the API service is run."
+              )
+@click.option('--userid',
+              'userid',
+              default=KFP_SDK_USERID,
+              help="your user ID (your ZG email)."
```

Comment:
This should not be ZG specific.  Let's just leave it as `your user ID (example: your email)`